bin/utils/convert-argv.js

```diff
@@ -6,7 +6,7 @@ const prepareOptions = require("./prepareOptions");
 const webpackConfigurationSchema = require("../config/webpackConfigurationSchema.json");
 const validateSchema = require("webpack").validateSchema;
 const WebpackOptionsValidationError = require("webpack").WebpackOptionsValidationError;
-const { webpackConfigPath } = require("@webpack-cli/utils/path-utils");
+const { webpackConfigPath } = require("../../packages/utils/path-utils");
```

Comment:
what should we do about this? 🤔

Reply:
Precisely, wondering :(

Reply:
This is the correct approach 

```js
const { webpackConfigPath } = require("@webpack-cli/utils/path-utils")
```

Does this create some problems?

Reply:
Yes, this is what I wrote when a large number of tests failed right before the last commit. @ematipico

Reply:
Before pushing the code, did you test it locally? Debug it locally might give more insights.

Probably when the code gets compiled by typescript, the function doesn't get exported correctly

Reply:
Look!! `@webpack-cli/utils` has to become a dependency of the main package :) That's the missing piece of the puzzle. We can't use a package if we don't add it to the `package.json` and we don't install it

Reply:
Hey @ematipico thanks for your findings, I did `npm i @webpack-cli/utils` in the project root to add it to deps, but I'm still seeing the same errors, wonder what we're missing 🤔

Reply:
Ah my bad, need to link the local version, on it

Reply:
In theory you don't need to do link but you have to run Lerna bootstrap. That's the correct way

Reply:
No help 🆘